.github/workflows/e2e-tests-ondemand.yaml

```diff
@@ -6,16 +6,49 @@ on:
       target:
         description: 'Target branch to run E2E tests over'
         required: true
-        # TODO: @andriisoldatenko revisit it
         default: 'release-1.1'
 
 permissions:
   checks: write
 
+# based on:
+# https://docs.dynatrace.com/docs/setup-and-configuration/technology-support/support-model-for-kubernetes
+# https://kubernetes.io/releases/
 jobs:
-  run-in-k8s-1-21:
-    name: Run in Kubernetes 1.21
+  # TODO: End of support: Aug 1, 2024, remove after this date
```

Comment:
this comment doesn't make any sense by itself. are you going to remove the whole k8s matrix?

Reply:
ooppsy, good catch

Reply:
@chrismuellner  fixed in 80809ff1